.env

```diff
@@ -0,0 +1,9 @@
+CAMUNDA_CONNECTORS_VERSION=0.17.1
+CAMUNDA_OPTIMIZE_VERSION=3.10.0-alpha5
+CAMUNDA_PLATFORM_VERSION=8.2.0-alpha5
+CAMUNDA_WEB_MODELER_VERSION=0.8.0-beta
+ELASTIC_VERSION=7.17.5
```

Comment:
Elasticsearch and Kibana had different versions before (`7.17.5` and `7.17.0`). I assume that running the same version should actually be more compatible and that it works as expected?

Reply:
I'm pretty sure it was just missed to update kibana while updating elastic 🙃 maybe I was even involved there at some point 🙈